OpenUtau/SimpleHelpers/FileEncoding.cs

```diff
@@ -0,0 +1,367 @@
+#region *   License     *
```

Comment:
Is this file a duplicate?

Reply:
Yes, I'm not sure why that got committed. Since it wasn't referenced in any of my changed code, I removed it.